.github/PULL_REQUEST_TEMPLATE/prometheusrules.md

```diff
@@ -0,0 +1,10 @@
+# Prometheus Rule change
```

Comment:
This template doesn't apply to every change in the repo. Not sure how better to handle this.

Reply:
Yeah I thought about it as well, that's why I went for creating a separate folder and a named template.
I was expecting this name to show up in the dropdown, when a template is picked